scripts/test.sh

```diff
@@ -13,6 +13,7 @@ cd "$(dirname $0)"
 ../contracts/context-config/build.sh
 ../contracts/proxy-lib/build-test-deps.sh
 ../contracts/icp/context-config/build.sh
-
+../contracts/icp/proxy-contract/build_mock_contracts.sh
+../contracts/icp/proxy-contract/build.sh
```

Comment:
`build_mock_contracts` also calls `proxy-contract/build.sh`, maybe we skip the bottom one?

Reply:
true, will fix it in final PR